WcaOnRails/app/views/registrations/_payment_form.html.erb

```diff
@@ -0,0 +1,48 @@
+<%= horizontal_simple_form_for :payment, url: competition_process_payment_path(@competition), html: { id: :form_payment } do |f| %>
+  <%= render 'entry_fee', label: "Paid", money_amount: @registration.paid_entry_fees %>
+  <%= f.input :stripeToken, as: :hidden, input_html: { id: :stripeToken } %>
+  <%= f.input :total_amount, as: :hidden, input_html: { id: :total_amount } %>
+  <%= render 'entry_fee', label: "Remaining", money_amount: @registration.outstanding_entry_fees %>
+  <% if @competition.enable_donations %>
+    <%= f.input :donation, as: :money_amount, currency: @registration.outstanding_entry_fees.currency.iso_code, value: "0", label: t('registrations.payment_form.labels.donation'), hint: t('registrations.payment_form.hints.donation') %>
+  <% end %>
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <%= f.button :submit, value: t('registrations.payment_button_text'), id: :donateButton, class: 'btn btn-primary' %>
```

Comment:
Is `donateButton` the right name here? This looks like more of a "payment button". Also, camelCase vs snake_case ids? I think snake_case is more consistent with the rest of this file and our codebase.